en/05_Contributing/13_CSS_Coding_Conventions.md

```diff
@@ -28,28 +24,27 @@ We use [Bootstrap 4](https://getbootstrap.com/) styles where possible.
 We follow the [AirBnB CSS Conventions](https://github.com/airbnb/css)
 and the [BEM](http://getbem.com/) methodology (block-element-modifier).
 
-Because we use [Bootstrap 4](https://getbootstrap.com/) which 
-does not follow [BEM](http://getbem.com/) naming convention there will be 
-a lot of places where class names voilate BEM. 
-However, please note that they are not a indicator of how to name classes. 
+Because we use [Bootstrap 4](https://getbootstrap.com/) which
+does not follow [BEM](http://getbem.com/) naming convention there will be
+a lot of places where class names voilate BEM.
+However, please note that they are not a indicator of how to name classes.
```

Comment:
```suggestion
Because we use [Bootstrap 4](https://getbootstrap.com/) which does not follow [BEM](http://getbem.com/) naming convention there will be a lot of places where class names voilate BEM. However, please note that they are not a indicator of how to name classes.
```

Reply:
I don't understand the need for this change, but it won't change anything so I'll do it lol

Reply:
Actually I won't. Look at the file as a whole. Making this change _here_ and not through the whole file will make it harder to maintain. Let's leave this sort of thing until we're ready to do https://github.com/silverstripe/developer-docs/issues/11